lib/swapi-node.js

```diff
@@ -101,7 +101,7 @@ module.exports = {
     },
     getStarship (options) {
         var opts = parseOptions(options);
-        return makeRequest('starship' + (opts.id ? '/' + opts.id : '/'));
+        return makeRequest('starships' + (opts.id ? '/' + opts.id : '/'));
```

Comment:
This is actually already in.  
https://github.com/salty-pig/swapi-node/commit/cb5f0abf7c8e216f4001fbc717c8d33e9ec733c5